substratevm/src/com.oracle.svm.core.posix/src/com/oracle/svm/core/posix/linux/LinuxSubstitutions.java

```diff
@@ -27,10 +27,16 @@
 import static com.oracle.svm.core.posix.headers.Time.gettimeofday;
 import static com.oracle.svm.core.posix.headers.linux.LinuxTime.CLOCK_MONOTONIC;
 import static com.oracle.svm.core.posix.headers.linux.LinuxTime.clock_gettime;
+import static org.graalvm.nativeimage.UnmanagedMemory.calloc;
+import static org.graalvm.nativeimage.UnmanagedMemory.free;
```

Comment:
We have changed styles and are now trying not to add more `import static` statements.  Also, since this is in POSIX code, you can just use `Libc.calloc` and `Libc.free`.